obstacle_rich_env/envs/robot.py

```diff
@@ -17,6 +19,9 @@ def make_dynamics(self):
             return dynamics_class(params=self._params, random_generator=self.random_generator)
         raise 'Robot dynamics not defined'
 
+    def mount_lidar(self, map, config):
+        self.lidar = Lidar(map, self, config)
```

Comment:
Please move lidar instantiation to Engine

Reply:
Lidar instantiation moved to the engine.